fbw-a380x/src/systems/instruments/src/PFD/DigitalAltitudeReadout.tsx

```diff
@@ -234,13 +234,13 @@ class Drum extends DisplayComponent<DrumProperties> {
             const digitRef = FSComponent.createRef<SVGTextElement>();
 
             if (this.props.type === 'hundreds') {
-                graduationElements.push(<text ref={digitRef} transform={`translate(0 ${offset})`} class={`FontLargest MiddleAlign ${this.color}`} x="11.631" y="7.1" />);
+                graduationElements.push(<text ref={digitRef} transform={`translate(0 ${offset})`} class={`FontMedium MiddleAlign ${this.color}`} x="13.1" y="6.8" />);
             } else if (this.props.type === 'thousands') {
-                graduationElements.push(<text ref={digitRef} transform={`translate(0 ${offset})`} class={`FontLargest MiddleAlign ${this.color}`} x="7.18" y="7.1" />);
+                graduationElements.push(<text ref={digitRef} transform={`translate(0 ${offset})`} class={`FontMedium MiddleAlign ${this.color}`} x="9.2" y="6.8" />);
             } else if (this.props.type === 'ten-thousands') {
-                graduationElements.push(<text ref={digitRef} transform={`translate(0 ${offset})`} class={`FontLargest MiddleAlign ${this.color}`} x="2.498" y="7.1" />);
+                graduationElements.push(<text ref={digitRef} transform={`translate(0 ${offset})`} class={`FontMedium MiddleAlign ${this.color}`} x="5.9" y="6.8" />);
             } else if (this.props.type === 'tens') {
-                graduationElements.push(<text ref={digitRef} transform={`translate(0 ${offset})`} class={`FontSmallest MiddleAlign ${this.color}`} x="4.5894" y="8.9133" />);
+                graduationElements.push(<text ref={digitRef} transform={`translate(0 ${offset})`} class={`FontTinyToSmallest MiddleAlign ${this.color}`} x="5.5" y="8.9133" />);
```

Comment:
I did a quick check and FontSmallest is still correct for the tens digits, for the others FontMedium seems to be correct

Reply:
Ok, reverted that as well. As a side note, I also activated the gear indication, see latest commit. I think using the LGCIU input is a good trade-off, as long as we don't have the LGCIS yet.